Models/User.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace WsuTriangle.AttendanceTracker.Models;
+
+public class User
+{
+    public int UserId { get; set; }
+    public string UserName { get; set; } = string.Empty;
+    public string PasswordHash { get; set; } = string.Empty;
```

Comment:
Do not store passwords in here. Instead, log into accounts via Microsoft, Google, etc etc. Emails will map to a user. Instead, store an email here.
```suggestion
```